### PR TITLE
[test-only] E2e. changing user

### PR DESCRIPTION
**Pull request by ScharfViktor**
created test for editing user
- [x] name
- [x] displayName
- [x] email
- [x] password
- [x] role

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/33582/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33582/tracing/user-should-be-able-to-read-and-dismiss-notifications-alice-2023-3-13-12-50-27.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33582/tracing/user-should-be-able-to-read-and-dismiss-notifications-brian-2023-3-13-12-50-33.zip 


</p></details>
:boom: The e2e-oc10 tests pipeline failed. The build has been cancelled.

File: tests/e2e/cucumber/steps/ui/adminSettings.ts [hunk 1]
```diff
@@ -242,3 +242,35 @@ When(
...
+    if (attribute === 'userName') {
+      const newUser = this.usersEnvironment.createUser({
+        key: value,
+        user: {
+          id: value,
+          displayName: '',
+          password: 'password',
+          email: ''
+        }
+      })
```
Comment (JammingBen) [inline tests/e2e/cucumber/steps/ui/adminSettings.ts:268]: Out of curiosity: why not call `this.usersEnvironment.getUser()` here instead?

**Review by JammingBen — APPROVED** [review]
LGTM 👍

File: tests/e2e/cucumber/steps/ui/adminSettings.ts (hunk 1 above)
Comment (ScharfViktor) [reply tests/e2e/cucumber/steps/ui/adminSettings.ts:268]: then we get error `user with key anna not found`. We need create `anna` in the store before deleting
https://github.com/owncloud/web/blob/master/tests/e2e/support/environment/userManagement.ts#L8-L9

**Pull request merged**